integration-tests/browser/src/index.ts

```diff
@@ -83,6 +83,10 @@ provider.start();
         throw new Error('Expected no orders to be rejected but got: ' + result.rejected.length);
     }
 
+    // Call getStatsAsync and make sure it works.
+    const stats = await mesh.getStatsAsync();
+    console.log(JSON.stringify(stats));
```

Comment:
Is this an adequate test or should we asset some of the returned values?

Reply:
Plan is to implement more robust tests after https://github.com/0xProject/0x-mesh/pull/649 is implemented. I've been talking to @jalextowle about specifics.

Reply:
@fabioberger mentioned this was okay in Slack. Since this was your only point of feedback I'm assuming it's okay to merge.